appenv/bin/gunicorn

```diff
@@ -0,0 +1,8 @@
+#!/Users/jormondroyd/Desktop/corndel_project/DevOps-Course-Starter/appenv/bin/python3
+# -*- coding: utf-8 -*-
+import re
+import sys
+from gunicorn.app.wsgiapp import run
+if __name__ == '__main__':
+    sys.argv[0] = re.sub(r'(-script\.pyw|\.exe)?$', '', sys.argv[0])
+    sys.exit(run())
```

Comment:
Should this file be checked in? It reads to me like you potentially installed this before you were managing Gunicorn with poetry, and I can't see any evidence that this is used anywhere